db/schema.rb

```diff
@@ -107,8 +107,18 @@
     t.integer "impressions_count", default: 0
   end
 
+  create_table "reads", force: :cascade do |t|
+    t.bigint "user_id", null: false
+    t.string "readable_type"
+    t.bigint "readable_id"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.index ["readable_type", "readable_id"], name: "index_reads_on_readable_type_and_readable_id"
+    t.index ["user_id"], name: "index_reads_on_user_id"
+  end
+
   create_table "solutions", force: :cascade do |t|
-    t.text "content", null: false
+    t.text "content"
```

Comment:
```suggestion
    t.text "content", null: false
```
`null: false` が取れてしまってます！

Reply:
ここ触ったつもりがないんですが、了解です！